transform/models/marts/repo_pushes_daily.sql

```diff
@@ -0,0 +1,12 @@
+Select distinct 
```

Comment:
Minor style: Let's choose either lower case or upper case for SQL keywords like `select`, `from`, etc.
And keep it consistent across all 4 models.